app/views/registrations/_form.haml

```diff
@@ -29,6 +29,8 @@
 
     - if AppConfig.settings.captcha.enable?
       = show_simple_captcha :object => 'user', :code_type => 'numeric'
+      :javascript
+        $('#user_captcha').addClass('form-control');
```

Comment:
* https://github.com/pludoni/simple-captcha/blob/master/lib/simple_captcha/view.rb#L46 ->
* https://github.com/diaspora/diaspora/blob/develop/app/views/simple_captcha/_simple_captcha.haml -> 
* https://github.com/pludoni/simple-captcha/blob/master/lib/simple_captcha/view.rb#L62 ->
* https://github.com/pludoni/simple-captcha/blob/master/lib/simple_captcha/view.rb#L94

so my guess is `= show_simple_captcha object: "user", code_type: "numeric", input_html: {class: "form-control"}`